assets/wizards/connections/views/main/webhooks.js

```diff
@@ -292,16 +292,18 @@ const Webhooks = () => {
 							description={ () => {
 								if ( endpoint.disabled && endpoint.disabled_error ) {
 									return (
-										__( 'This endpoint is disabled due excessive request errors: ', 'newspack' ) +
-										endpoint.disabled_error
+										__(
+											'This endpoint is disabled due excessive request errors: ',
```

Comment:
typo: 'This endpoint is disabled due **to** excessive request errors'

Reply:
Thank you for catching that! Updated in b4ff2bdab6eeeeab77f7795c40069d5396c5c282 and 41494292d41dd33e1ad5fd393e790ea3337b0e2a to remove the colon from the localized string.